StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -372,6 +414,34 @@ extension SavedPaymentMethodCollectionView {
                 }()
             }
         }
+
+        private func activateDefaultBadge() {
+            NSLayoutConstraint.deactivate([
+                labelBottomConstraint
+            ])
+            NSLayoutConstraint.activate([
+                labelHeightConstraint
+            ])
+            defaultBadgeConstraints = [
+                defaultBadge.topAnchor.constraint(
+                    equalTo: label.bottomAnchor, constant: 4),
+                defaultBadge.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+                defaultBadge.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 2),
+                defaultBadge.trailingAnchor.constraint(equalTo: contentView.trailingAnchor)
+            ]
+            NSLayoutConstraint.activate(defaultBadgeConstraints)
+        }
+
+        private func deactivateDefaultBadge() {
+            NSLayoutConstraint.deactivate(defaultBadgeConstraints)
+            NSLayoutConstraint.deactivate([
+                labelHeightConstraint
+            ])
+            NSLayoutConstraint.activate([
+                labelBottomConstraint
+            ])
+        }
```

Comment:
Naming on these functions feels awkward since we aren't activating a badge.  Maybe a better name might be "activateConstraintsDefaultBadge" and "deactivateConstraintsforDefaultBadge"?